spec/betterc.dd

```diff
@@ -0,0 +1,47 @@
+Ddoc
+
+$(SPEC_S Better C,
+
+`-betterC` is a Command-Line flag for `dmd`, which generates light-weight,
+Druntime-free binaries with are suitable for linking from C.
```

Comment:
Sorry nitpick again about this paragraph, but I had a bit more time to think about it and I came up with this:

> `-betterC` is a command-line flag for `dmd`, which restricts the compiler's support of certain runtime features. Notably, D programs or libraries would not be linked with Druntime. The use of compile-time features is not restricted in any way. Limiting a program or library to this subset of runtime features is useful when targeting constrained environments where the use of such features is not practical or possible.
>
> Additionally, this is also makes embedding D libraries in larger projects easier by:
> * Simplifying the process of integration at the build-system level
> * Removing the need  to ensure that Druntime is properly initialized on calls to the library, when an initialization step is not performed before the library is used.
> * Mixing memory management strategies (GC + manual memory management) can sometimes be tricky, hence removing D's GC from the equation may be good solution

Most probably this can be made more clear or informative, so feel free to improve it.